config/resources/besluit-domain.lisp

```diff
@@ -200,6 +200,8 @@
                                         :as "kind")
              (meeting                   :via      ,(s-prefix "dct:isPartOf")
                                         :as "main-meeting")
+             (minutes                   :via      ,(s-prefix "besluitvorming:heeftNotulen")
```

Comment:
Is `besluitvorming:heeftNotulen` effectief een ding? Ik kan precies enkel [besluit:heeftNotulen](https://data.vlaanderen.be/doc/vocabularium/besluit/#heeftNotulen) vinden ...

Reply:
Ik heb dat overgenomen uit het ticket maar is mss een oversight van @tdn ?

Reply:
https://data.vlaanderen.be/doc/applicatieprofiel/besluitvorming/#Vergaderactiviteit